test/skate_web/controllers/detour_route_controller_test.exs

```diff
@@ -0,0 +1,42 @@
+defmodule SkateWeb.DetourRouteControllerTest do
+  use SkateWeb.ConnCase
+  import Test.Support.Helpers
+
+  setup %{conn: conn} do
+    {:ok, conn: put_req_header(conn, "accept", "application/json")}
+  end
+
+  describe "directions" do
+    setup do
+      bypass = Bypass.open()
```

Comment:
**issue (non-blocking):** if this test works I think it's fine, but just a note that Bypass seems pretty much unmaintained at this point, so we should probably try to use alternatives going forward.

Reply:
Yup! I'm working on replacing it with Mox, per _all the recommendations_